cfme/tests/cli/test_appliance_console.py

```diff
@@ -1352,7 +1353,21 @@ def test_appliance_console_datetime_negative():
             3.
             4. Invalid date/time should not be applied, check for failure there.
     """
-    pass
+
+    if invalid == "invalid_date":
+        invalid_dates = ['202020-12-3', '2020-13-3', '2020-12-35']
+        cmd_sets = [("ap", RETURN, "3", "Y", date) for date in invalid_dates]
+    else:
+        invalid_times = ['25:13:01', '22:61:01', '24:11:61']
+        cmd_sets = [("ap", RETURN, "3", "Y", str(fauxfactory.gen_date()), time)
+                    for time in invalid_times]
+
+    for cmd in cmd_sets:
+        # TODO(dgaikwad): remore if condition once issue got fixed
+        if "2020-12-35" != cmd[4] and BZ(1790311, forced_streams=["5.10", "5.11"]).blocks:
+            result = appliance.appliance_console.run_commands(cmd, timeout=30, output=True)
+            assert "Please provide in the specified format" in result[-1], (
```

Comment:
@dgaikwad This BZ(https://bugzilla.redhat.com/show_bug.cgi?id=1790311) is **CLOSEDWONTFIX**. 
Can you please reopen or create new BZ for this issue?
I can see your comments as this issue is still reproducible. So I think you can reopen this one or report new one.

Reply:
Any update on this query?

Reply:
We have removed this condition now